CONTRIBUTING.md

```diff
@@ -0,0 +1,246 @@
+
+Review our [Security
+Policy](https://github.com/Appsilon/.github/blob/main/SECURITY.md). **Do not**
+file a public issue for security vulnerabilities.
+
+### :beetle: Bug Reports and Other Issues
+
+A great way to contribute to the project is to send a detailed issue when you
+encounter a problem. We always appreciate a well-written, thorough bug report.
+
+In short, since you are most likely a developer, **provide a ticket that you
+would like to receive**.
+
+- **Review the documentation and [Support
+  Guide](https://github.com/Appsilon/.github/blob/main/SUPPORT.md)** before
+  opening a new issue.
+
+- **Do not open a duplicate issue!** Search through existing issues to see if
+  your issue has previously been reported. If your issue exists, comment with
+  any additional information you have. You may simply note "I have this problem
+  too", which helps prioritize the most common problems and requests.
+
+- **Prefer using
+  [reactions](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)**,
+  not comments, if you simply want to "+1" an existing issue.
+
+- **Fully complete the provided issue template.** The bug report template
+  requests all the information we need to quickly and efficiently address your
+  issue. Be clear, concise, and descriptive. Provide as much information as you
+  can, including steps to reproduce, stack traces, compiler errors, library
+  versions, OS versions, and screenshots (if applicable).
+
+- **Use [GitHub-flavored
+  Markdown](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax).**
+  Especially put code blocks and console outputs in backticks. This improves
+  readability.
+
+## :love_letter: Feature Requests
+
+Feature requests are welcome! While we will consider all requests, we cannot
+guarantee your request will be accepted. We want to avoid [feature
+creep](https://en.wikipedia.org/wiki/Feature_creep). Your idea may be great,
+but also out-of-scope for the project. If accepted, we cannot make any
+commitments regarding the timeline for implementation and release. However, you
+are welcome to submit a pull request to help!
+
+- **Do not open a duplicate feature request.** Search for existing feature
+  requests first. If you find your feature (or one very similar) previously
+  requested, comment on that issue.
+
+- **Fully complete the provided feature request template.** The feature request template
+  asks for all necessary information for us to begin a productive conversation.
+
+- Be precise about the proposed outcome of the feature and how it relates to
+  existing features. Include implementation details if possible.
+
+## :mag: Triaging Issues
+
+You can triage issues which may include reproducing bug reports or asking for
+additional information, such as version numbers or reproduction instructions.
+Any help you can provide to quickly resolve an issue is very much appreciated!
+
+## :repeat: Submitting Pull Requests
```

Comment:
Do we have a template to write Pull Request names like with commit names?

Like e.g. capitalize and do not end with a dot

Reply:
It would be difficult to impose specific pull request title format considering various tools / integrations (e.g. Rennovate / Bolt) submit PRs with their own format:

![Screenshot 2022-03-10 at 11 40 07](https://user-images.githubusercontent.com/9083008/157644927-296a373e-0308-47f7-8299-24ad006b09c9.png)
![Screenshot 2022-03-10 at 11 40 34](https://user-images.githubusercontent.com/9083008/157644994-751d19c5-90fd-43fd-a95b-47a9e17b970f.png)

Reply:
btw, when merging using squash and merge, it overwrites the commits with a single commit, and by default, it uses the name of the PR and it looks bad when it's different than the conventional commit one.

This is a separate thing, and I agree with your point regarding the PR title format.

Reply:
Merge strategies are also a valid point for discussion and whether there should be an organization-wide policy for that. Perhaps we can leave it as is for now.

Both `squash and merge` and `regular merge` will create a master commit with all commit titles listed:
![Screenshot 2022-03-10 at 12 01 20](https://user-images.githubusercontent.com/9083008/157648728-bded89db-0ef2-440e-97e0-543085bdc953.png)

![Screenshot 2022-03-10 at 11 54 50](https://user-images.githubusercontent.com/9083008/157647542-caee988a-35cb-4376-a2ff-ccfac0b1599e.png)

Reply:
I was refering to this case. When I by mistake merged with the name of th PR and now it looks like this:
<img width="880" alt="Screenshot 2022-03-11 at 09 07 40" src="https://user-images.githubusercontent.com/46781099/157827578-85e7baf1-deb5-41f4-8b83-9ae016f8f250.png">